modules/indicia_svc_import/controllers/services/import.php

```diff
@@ -654,6 +654,16 @@ public function upload() {
           }
         }
 
+        // Special case for samples: the sref system is fixed for the file, but is not required in location_id provided
```

Comment:
There looks like there is a typo here - sef rather than sref on line 663 and 664.

Trying to understand this - is it the case the sref system will be obtained from the location centroid?

Reply:
Yes there was a typo - should have been corrected in the last commit https://github.com/Indicia-Team/warehouse/pull/346/commits/9e8e1d046cfb2c535363abe25a90897ab83a84fc.  
When a location is provided, the sample model validation expects either both the entered_sref and entered_sref_system to be provided, or neither: I assume in this case, all this info is provided by the location record.  This change allows the importer to specify a location id, but no entered_sref, as allowed by the model - the point being that the importer sets the system whether or not the entered sref is provided or not, hence this change